### PR TITLE
[DOC]Fix Architecture

### PR DESCRIPTION
**Pull request by zwangsheng**
zwangsheng [Closes #446](https://github.com/yaooqinn/kyuubi/pull/446)    Target Issue [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_

Fix the problem with the display of the article

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/overview/architecture.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The only heavyweight case is when there is no `SparkContext` instantiated or cac
 which usually happens when the user is connecting for the first time or has not connected for a long time.
 This one-time cost session maintenance model can meet most of the ad-hoc fast response requirements.
 
-Kyuubi maintains connections to `SparkConext` in a loosely coupled fashion. These `SparkContext's can be Spark programs created locally in client deploy mode by this service instance,
+Kyuubi maintains connections to `SparkConext` in a loosely coupled fashion. These `SparkContext` can be Spark programs created locally in client deploy mode by this service instance,
```
Comment (yaooqinn) [inline docs/overview/architecture.md:52]: the `s` is needed

**Review by yaooqinn — APPROVED** [review]

File: docs/overview/architecture.md (hunk 1 above)
Comment (zwangsheng) [reply docs/overview/architecture.md:52]: But many `SparkContext` without s in other place of architecture.md

Comment (yaooqinn) [reply docs/overview/architecture.md:52]: Then` these `should be `the`

Comment (yaooqinn) [reply docs/overview/architecture.md:52]: `a Spark program`

**Comment by yaooqinn** [comment]
thanks, merged to master for v1.2.0

**Pull request closed without merging**